static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -48,7 +50,10 @@ function Breadcrumbs({frames, startTimestampMs}: Props) {
   // re-render when items are expanded/collapsed, though it may work in state as well.
   const expandPathsRef = useRef(new Map<number, Set<string>>());
 
-  const deps = useMemo(() => [frames], [frames]);
+  const filterProps = useBreadcrumbFilters({frames: frames || []});
+  const {items} = filterProps;
+
+  const deps = useMemo(() => [items], [items]);
```

Comment:
We'll need some more things below, i always just look at the other tabs to see what needs to be copy+pasted. I'm sure there's a way to refactor so it's less copy/paste, but for now idk...


```suggestion
  const {items, searchTerm, setSearchTerm} = filterProps;
  const clearSearchTerm = () => setSearchTerm('');
  
  const deps = useMemo(() => [items, searchTerm], [items, searchTerm]);
```